vm/compiler/src/ledger/mod.rs

```diff
@@ -317,9 +248,27 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Ledger<N, B, P> {
 
     /// Returns a candidate for the next block in the ledger.
     pub fn propose_next_block<R: Rng + CryptoRng>(&self, private_key: &PrivateKey<N>, rng: &mut R) -> Result<Block<N>> {
-        // TODO (raychu86): Add logic to transaction selection from the mempool.
         // Construct the transactions for the block.
-        let transactions = self.memory_pool.values().collect::<Transactions<N>>();
+        let transactions = {
+            // TODO (raychu86): Add more sophisticated logic for transaction selection.
```

Comment:
I wonder if we should start tracking this sort of future work as github issues instead of just todo comments